### ISSUE TITLE
"Add expence" button isn't accessible

### ISSUE THREAD

**Issue by limonte** [vaadin/expense-manager-demo#91]
`aria-label` is missing for `#add-button`, it's pronounced as "clickable button" by screen readers.
